### PR TITLE
feat(server): add append step

### PR DESCRIPTION
**Pull request by fspot**

### CODE REVIEW

File: server/weaverbird/steps/append.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    def execute(
+        self,
+        df: DataFrame,
+        domain_retriever,
+        execute_pipeline: PipelineExecutor,
+    ) -> DataFrame:
+        for pipeline in self.pipelines:
+            other_df = execute_pipeline(pipeline)
+            df = df.append(other_df, ignore_index=True)
```
Comment (davinov) [inline server/weaverbird/steps/append.py:24]: You can directly append a list of dataframes:

```python
In [1]: import pandas as pd

In [2]: df = pd.DataFrame([[1, 2], [3, 4]], columns=list('AB'))

In [3]: df2 = pd.DataFrame([[5, 6], [7, 8]], columns=list('AB'))

In [4]: df3 = pd.DataFrame([[9, 10], [11, 12]], columns=list('AB'))

In [5]: df.append([df2, df3])
Out[5]: 
    A   B
0   1   2
1   3   4
0   5   6
1   7   8
0   9  10
1  11  12
```

Comment (davinov) [inline server/weaverbird/steps/append.py:24]: You can directly append a list of dataframes:

```python
In [1]: import pandas as pd

In [2]: df = pd.DataFrame([[1, 2], [3, 4]], columns=list('AB'))

In [3]: df2 = pd.DataFrame([[5, 6], [7, 8]], columns=list('AB'))

In [4]: df3 = pd.DataFrame([[9, 10], [11, 12]], columns=list('AB'))

In [5]: df.append([df2, df3])
Out[5]: 
    A   B
0   1   2
1   3   4
0   5   6
1   7   8
0   9  10
1  11  12
```

**Review by davinov — COMMENTED** [review]
LGTM, maybe you can simplify a bit but that's all!:

File: server/weaverbird/steps/append.py (hunk 1 above)
Comment (fspot) [reply server/weaverbird/steps/append.py]: feacdde

**Review by davinov — APPROVED** [review]

**Pull request merged**